tests/unit/Scaffold/Lock/LockFileTest.php

```
@@ -199,6 +201,35 @@ public function testReadThrowsOnMalformedJson(): void
 
+    public function testReadThrowsWhenFileGetContentsFails(): void
+    {
+        $path = "{$this->tempDir}/scaffold-lock.json";
+        file_put_contents($path, '{"providers":{},"files":{}}');
+
+        MockerState::addCondition(
+            'yii\\scaffold\\Scaffold\\Lock',
+            'file_get_contents',
+            [$path, false, null, 0, null],
+            false,
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xepozz\InternalMocker\MockerState' in method 'testReadThrowsWhenFileGetContentsFails'. (undefined)

(StaticAccess)